test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -100,13 +104,22 @@ namespace iroha {
       }
 
       /**
-       * Check that command result contains specific error code
+       * Check that command result contains specific error code and error
+       * message
        * @param cmd_result to be checked
        * @param expected_code to be in the result
+       * @param expected_substrings - collection of strings, which are expected
+       * to be in command error
        */
-#define CHECK_ERROR_CODE(cmd_result, expected_code) \
-  ASSERT_TRUE(err(cmd_result));                     \
-  ASSERT_EQ(err(cmd_result)->error.error_code, expected_code);
+#define CHECK_ERROR_CODE_AND_MESSAGE(                \
```

Comment:
Why make it a macro? Since there is only one such check in each test, if you make it a function, it will still be clear which test has failed.

Reply:
We agreed in previous PR, which was about introducing the error codes, that this is an acceptable way. Also, there is one test, `TransferAccountAssetTest.NoAccount`, which does two calls to this macro